recipes-debian/wireless-tools/wireless-tools_debian.bb

```diff
@@ -0,0 +1,51 @@
+# base recipe: meta/recipes-connectivity/wireless-tools/wireless-tools_30.pre9.bb
+# base branch: warrior
+# base commit: f7137f94905c0d0ef1c0af8e53db18de00a17fec
```

Comment:
The wireless-tools was removed in Yocto/OE because of its obsoletion.
Why don't you use `iw`?